### ISSUE TITLE
Regarding Subscription Workflow

### ISSUE THREAD

**Issue by knit-pay** [wp-pay/core#42]
Hello team

Generally, there are 2 type of subscription methods, which most of the payment gateways use. Which one out of these two, wp-pay supports? does it supports both of them or only one? If it supports both of them, are you using any parameter to control which method to use?
1. Merchant/Seller setup subscription directly on payment gateway dashboard. After that payment gateway charges customer directly on each billing cycle. Seller don't need to ask payment gateway to charge on each billing cycle. Payment gateway handle it on their own.
2. Payment gateway don't charge automatically on each billing cycle, Merchant/Seller has to do the API call to Payment Gateway to make a charge on each billing cycle.

**Comment by rvdsteege** [comment]
We're using the 2nd method with https://github.com/wp-pay-gateways/mollie, so that we have full control over when payments are created from within WordPress.

**Comment by knit-pay** [comment]
Thanks for the update. Are you planning to add support for the first method also in the future?

**Comment by rvdsteege** [comment]
No plans currently, but I can imagine we would need to make some changes when we want to add support for recurring payments for a gateway which only works with the 1st method.

We started a while ago with recurring payments in https://github.com/wp-pay-gateways/mollie, which actually supports both methods but we decided on using the 2nd method. At the moment, that is our only gateway which supports recurring payments. We've optimised a lot, before we're expanding the support to other gateways.
